packages/jazz-tools/src/tests/ContextManager.test.ts

```diff
@@ -26,6 +26,7 @@ import {
   setupJazzTestSync,
 } from "../testing";
 
+// @ts-ignore Typescript in VSCode doesn't like top level await
```

Comment:
```suggestion
// @ts-expect-error Typescript in VSCode doesn't like top level await
```
so the day vs code is happy we notice, this disable altogether

Reply:
The problem is that tsc doesn't consider it an error so `@ts-expect-error` makes the typecheck fail

Reply:
ah good point, nvm then